module/documents/activity/mixin.mjs

```diff
@@ -425,6 +426,21 @@ export default Base => class extends PseudoDocumentMixin(Base) {
 
   /* -------------------------------------------- */
 
+  /**
+   * Apply any keybindings that might affect usage process.
+   * @param {ActivityUseConfiguration} config       Configuration info for the activation.
+   * @param {ActivityDialogConfiguration} dialog    Configuration info for the configuration dialog.
+   * @param {ActivityMessageConfiguration} message  Configuration info for the chat message created.
+   * @protected
+   */
+  _applyKeybindings(config, dialog, message) {
+    dialog.configure ??= !areKeysPressed(config.event, "skipDialogNormal")
+      && !areKeysPressed(config.event, "skipDialogAdvantage")
+      && !areKeysPressed(config.event, "skipDialogDisadvantage");
```

Comment:
Let's just use SHIFT for now, and keep the other modifiers in reserve, I could imagine them having some use in certain consumption contexts.

Reply:
The trouble is, if we don't handle the other buttons then you won't be able to skip the usage dialog and roll the subsequent primary action with advantage/disadvantage, because that initial event is what is forwarded to the primary action.

Reply:
I think we need to reconsider then, since we've overloaded the SHIFT modifier here. The control hint is that you can SHIFT+Click to skip the activity selection dialog, but actually, with these changes, you not only skip activity selection, but you also skip consumption, and you force the resulting attack roll from #4555 to be a normal roll, no?

Having the modifier fast-forward the entire process is *probably* what we want, but I need to think of some way to communicate that more clearly in the control hint. Let me know if you have any suggestions.